depthai_nodes/ml/parsers/yolo.py

```diff
@@ -107,10 +187,39 @@ def setNumKeypoints(self, n_keypoints):
         """
         self.n_keypoints = n_keypoints
 
+    def setAnchors(self, anchors):
+        """Sets the anchors for the YOLO model.
+
+        @param anchors: The anchors for the YOLO model.
+        @type anchors: List[np.ndarray]
+        """
+        self.anchors = anchors
+
+    def setYoloVersion(self, yolo_version):
+        """Sets the version of the YOLO model.
+
+        @param yolo_version: The version of the YOLO model.
+        @type yolo_version: YOLOVersion
+        """
+        try:
+            self.yolo_version = YOLOVersion(yolo_version.lower())
+        except ValueError as err:
+            raise ValueError(
+                f"Invalid YOLO version {yolo_version}. Supported YOLO versions are {[e.value for e in YOLOVersion][:-1]}."
+            ) from err
+
+    def setOutputLayerNames(self, output_layer_names):
```

Comment:
add types to function and what it returns: 
`def setOutputLayerNames(self, output_layer_names: List[str]) -> None:`

Reply:
done